src/web/services/account.service.ts

```diff
@@ -0,0 +1,61 @@
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { AccountInfo, AdminSearchResult, JoinLink, MessageOutput } from '../types/api-output';
+import { AccountCreateRequest } from '../types/api-request';
+import { HttpRequestService } from './http-request.service';
+
+/**
+ * Handles account related logic provision
+ */
+
+@Injectable({
+  providedIn: 'root',
+})
+
+export class AccountService {
```

Comment:
You can use `ng g s Account` so that Angular CLI will generate corresponding test file also. Instead of copy here.